metals/src/main/scala/scala/meta/internal/builds/BuildTools.scala

```diff
@@ -121,21 +121,40 @@ final class BuildTools(
   )
   def isBazel: Boolean = bazelProject.isDefined
 
+  private def customProjectRoot =
+    userConfig().customProjectRoot
+      .map(_.trim() match {
+        case "." => workspace
+        case relativePath => workspace.resolve(relativePath)
```

Comment:
Just to make sure, if customProjectRoot is empty (`Some("")`) it resolves to workspace here and works as if it was `None`?

Reply:
Empty is filtered out while `fromJson` on the whole `UserConfig`.